src/Microsoft.Data.Analysis/PrimitiveDataFrameColumn.cs

```diff
@@ -20,7 +20,7 @@ namespace Microsoft.Data.Analysis
     public partial class PrimitiveDataFrameColumn<T> : DataFrameColumn, IEnumerable<T?>
         where T : unmanaged
     {
-        private PrimitiveColumnContainer<T> _columnContainer;
+        internal PrimitiveColumnContainer<T> _columnContainer;
```

Comment:
I'm not a huge fan of making fields exposed outside of a class. If we really need to get at this from outside the class, can we make an `internal` property to expose it instead?